probe-rs/src/core/mod.rs

```diff
@@ -341,11 +342,13 @@ pub enum SpecificCoreState {
     /// The state of an ARMv6-M core.
     Armv6m(State),
```

Comment:
I guess we should rename this old state struct to `CortexMState` :)

Reply:
Yeah, much cleaner.  Eventually I will find and fix all the Cortex-M assumptions in this code...